todo_app/data/trello_items.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+from telnetlib import STATUS
+from turtle import update
+import requests
+import os
+import json
+from flask import Flask, render_template, request, redirect, session
+
+app = Flask(__name__)
+app = Flask(__name__, template_folder='../templates')
+
+os.environ['TRELLO_TOKEN'] = "9cc5e1de416f419f2f1b3f5effda1d8bd01fde1e2f7291d656629b05633c708c"
```

Comment:
This value is a secret; with access to it anybody can interact with your Trello board! We should treat this as compromised since it has been stored in your code, and revoke this token in Trello now, and if we bring this functionality in line with the original `app.py` file then we can just store this value in the `.env` file on your computer that doesn't get committed to Git.